src/components/views/spaces/SpaceSettingsVisibilityTab.tsx

```diff
@@ -84,14 +89,9 @@ const SpaceSettingsVisibilityTab = ({ matrixClient: cli, space, closeSettingsFn
                     <br />
                     { _t("This may be useful for public spaces.") }
                 </p>
-            </>;
-        } else {
-            advancedSection = <>
-                <AccessibleButton onClick={toggleAdvancedSection} kind="link" className="mx_SettingsTab_showAdvanced">
-                    { _t("Show advanced") }
-                </AccessibleButton>
-            </>;
-        }
+            </div>
+            }
```

Comment:
The offset looks a bit awkward here:
```suggestion
            </div> }
```